shell/components/LocaleSelector.vue

```diff
@@ -50,13 +61,28 @@ export default {
 <template>
   <div>
     <div v-if="mode === 'login'">
-      <div v-if="showLocale">
+      <div
+        v-if="showLocale"
+        role="menu"
+        :aria-label="t('locale.menu')"
+        class="locale-login-container"
+        tabindex="0"
+        @click="openLocaleSelector"
+        @blur.capture="closeLocaleSelector"
+        @keyup.enter="openLocaleSelector"
+        @keyup.space="openLocaleSelector"
+      >
         <v-dropdown
```

Comment:
I guess the keyboard behaviour for the menu will be fixed when we wire in a standard menu component, as Phil notes. Tab should not move focus between menu items - the up/down arrow keys should do that - but we can tackle that =in the standard menu component. We will need to make sure we wire it in here.

Reply:
Me and @rak-phillip agreed that he would work on his new menu component and will replace all instances on the code afterwards.

The menu is now browsable with the `tab` key despite not being optimal. It's a definite improvement but not the final solution.